src/setup/rgw_setup

```diff
@@ -34,6 +37,22 @@ class SetupCmdBase(Cmd):
         parser.add_argument('--services', default='services', help='services')
         parser.add_argument('--config', default='config_url', help='config')
 
+    def get_cortx_config(self, cortx_config_url: str):
```

Comment:
This is not an interface of the class hence must be _()
`def _get_cortx_conf(self, cortx_config_url: str):`

Reply:
done.